.github/workflows/ci.yml

```diff
@@ -391,3 +404,29 @@ jobs:
           name: ${{ matrix.name }}_report
           path: |
             /tmp/report/
+            
+  check_driver_schema:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    needs: paths-filter
+    if: needs.paths-filter.outputs.driver_events_changed == 'true' && needs.paths-filter.outputs.driver_schema_changed == 'false'
+    steps:
+      - name: Check driver SCHEMA_VERSION
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: |
```

Comment:
I am not sure whether to make the job fail too (`exit 1`)

Reply:
I think the comment is enough, at the very least it should be a red flag for maintainers on its own.